### PR TITLE
Rename KeybaseDaemon to KeybaseService

### PR DESCRIPTION
**Pull request by gabriel**
The interface name `KeybaseDaemon` implies that it is a daemon, when it may not be. I think `KeybaseService` would be more appropriate?

The default impl for desktop is a daemon RPC style service but it could be implemented without 
daemon or RPC (as it is on mobile).

### CODE REVIEW

**Comment by gabriel** [comment]
BTW, if you'd rather not do the rename, no biggie, can close... my naming OCD was strong today

**Comment by gabriel** [comment]
@strib

File: libfuse/mount_test.go [hunk 1]
```diff
@@ -395,7 +395,7 @@ func TestReaddirPublic(t *testing.T) {
 }
 
 type kbdaemonBrokenIdentify struct {
-	libkbfs.KeybaseDaemon
+	libkbfs.KeybaseService
```
Comment (akalin-keybase) [inline libfuse/mount_test.go]: kbdaemon -> kbservice

**Comment by akalin-keybase** [comment]
lgtm

**Pull request merged**